modules/apiserversource-yaml.adoc

```diff
@@ -236,10 +273,34 @@ Data,
 [id="delete-apiserversource-yaml_{context}"]
 == Deleting the ApiServerSource
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `oc` commands:
```

Comment:
This is a good introduction for the procedure that follows. I would keep it and add a .Procedure title afterwards before the steps.